english/crouch-dragon/crouch-dragon.md

```diff
@@ -0,0 +1 @@
+Once upon a time, there is a baby dragon living in the sea. The baby dragon is very smart and nice, but some devils curse it. It lost its father, entwined into a family turnmoil, and then lost its beloved elder brother. It seemed to be doomed to live a miseral life. Nonetheless, with great perserverance, it rebounced back after crouching for a while. After having been surviving all the disasters and unfortune, it grew up with formidable willpower and undefeatable inner spirit. Here comes it, up front in the face of so-called destiny.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.